hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/ConfDumper.java

```diff
@@ -25,26 +25,32 @@
 
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
 
 import com.baidu.hugegraph.config.HugeConfig;
 import com.baidu.hugegraph.config.OptionSpace;
 import com.baidu.hugegraph.config.TypedOption;
 import com.baidu.hugegraph.dist.RegisterUtil;
+import com.baidu.hugegraph.server.RestServer;
 import com.baidu.hugegraph.util.E;
+import com.baidu.hugegraph.util.Log;
 
 public class ConfDumper {
 
     public static final String EOL = System.getProperty("line.separator");
 
+    private static final Logger LOG = Log.logger(RestServer.class);
```

Comment:
use `RestServer` name for the class's log?

Reply:
chage to ConfDumper